launch_checklist.module

```diff
@@ -117,5 +138,572 @@ function launch_checklist_checklistapi_checklist_items() {
         '#description' => t('Ensure Spam prevention measures are in place.'),
       ],
     ],
+
+    // Section 4 SEO.
```

Comment:
These lists are pretty long.  It would be great to break out the sections into their own .inc files.

What would be the best way to do that?